utils/prepare-commit-msg.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# This script is intended to be used a a git hook and will prepend the ticket
+# number to a commit message in the correct format for Github/Gitlab to parse.
```

Comment:
Can we add something like `(use of this is optional and is meant to be helpful)` at the top or something (just thinking of the beginner contributor who might stumble upon this and think it's something they HAVE to do.